.github/workflows/MCLanguageFileHelper.yml

```
@@ -0,0 +1,19 @@
+            - name: MC Language File Helper
```

actionlint (1.7.7)

workflow is sequence node but mapping node is expected

(syntax-check)